en-US/snippets/2017/oct2017.lang

```diff
@@ -0,0 +1,99 @@
+## NOTE: Reference bug 1408280. Deadline: 25 October, 2017.
+## NOTE: Guidelines on localizing snippets: https://goo.gl/nEhdt4. The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: No character length limit, but try not to exceed English length.
+
+
+# Snippet for Firefox quantum
+# DeLorean refers to the time machine from the Back to the Future films. If there are better known time machine reference in your culture, feel free to swap it out.
+# Link: https://www.mozilla.org/firefox/quantum/
```

Comment:
Either this link is wrong, or they are pointint 7 out of 8 languages to a page available only in English. The new Quantum landing page will be at /firefox, and available only on Nov 14.

Reply:
Confirmed to change it to landing page instead https://www.mozilla.org/firefox/